ci/unit_tests/workspace_addjson/main_test.py

```diff
@@ -0,0 +1,58 @@
+
+import os, unittest, sys, codecs, json
+
+import workspace_addjson
+
+class TestMain(unittest.TestCase):
+
+    def test_addingJsonDictionary(self):
+        ''' Tests for including additional json dictionary to arbitrary location in dialog nodes '''
+
+        cmd = 'cp ./ci/unit_tests/workspace_addjson/main_data/workspace.json ./ci/unit_tests/workspace_addjson/main_data/workspace_forAddJsonTest.json'
+        os.system(cmd)
+
+        workspaceFile = 'workspace_forAddJsonTest.json'
+        directory = './ci/unit_tests/workspace_addjson/main_data/'
+        jsonAddPath = './ci/unit_tests/workspace_addjson/main_data/jsonToAdd.json'
+        targetNode = 'parent'
```

Comment:
targetNode -> targetKey... we actually do not replace any node but key... it should be also change in comments and usage

Reply:
fixed